shell/platform/android/io/flutter/embedding/android/FlutterImageView.java

```diff
@@ -99,6 +99,7 @@ private static void logW(String format, Object... args) {
   }
 
   @TargetApi(19)
+  @SuppressLint("WrongConstant") // RGBA_8888 is a valid constant.
```

Comment:
Looks like you are forgetting to import `android.annotation.SuppressLint`

Reply:
🤦‍♂️ done